README.md

```diff
@@ -0,0 +1,12 @@
+## Hangman
+Play hangman on the Ethereum network. Hangman is a word guessing game the word completely or guess letters the word contains. Each try uses an attempt and players win when guessed correctly and lose when all attempts have been used. Random word is fetched from the [Wikipedia](Wikipedia.org) API via Chainlink
```

Comment:
Something like the following:

```
Play hangman on the Ethereum network

Hangman is a game where a player must guess the word in the allotted guesses. 
Players are allowed to guess letters and words. 
Guessing a word or letter incorrectly uses an attempt. 
When all attempts are used or if the word is identified the game ends.
```  

Take a look at what Wikipedia says. 

Also there should be another section that talks about how it works on a tech level. It doesn't need to be in first paragraph.